web-common/src/features/dashboards/time-controls/DefaultTimeRangeMenuItem.svelte

```diff
@@ -1,15 +1,15 @@
 <script lang="ts">
   import { humaniseISODuration } from "@rilldata/web-common/lib/time/ranges/iso-ranges";
-  import MenuItem from "../../../components/menu/core/MenuItem.svelte";
+  import DropdownMenuItem from "@rilldata/web-common/components/dropdown-menu/DropdownMenuItem.svelte";
 
   export let selected = false;
   export let isoDuration: string;
 </script>
 
-<MenuItem animateSelect={true} on:select>
+<DropdownMenuItem on:click on:on:mouseenter>
```

Comment:
I've never seen this `on:on:*` syntax. Is this a typo? I tried googling for it but couldn't find anything about it, and when I remove one of the `on:`s the behavior seems the same, so kind of assume it's a typo.